### ISSUE TITLE
WordAds: more options to manage ads

### ISSUE THREAD

**Issue by ricjcs** [Automattic/jetpack#18612]
The options for managing ads are few. It would be nice if we had more options, such as:

**Turn on/off the ads inside the post and pages.**
Wordads sometimes adds advertisements inside the posts. But to turn off these ads it is necessary to turn off the ones below the posts. You cannot turn on or off exclusively those that appear inside the posts. These ads also sometimes appear, sometimes not. There should be an option to manage that.

**Options for ads with different sizes.**
For example the standard square ad has the dimension of 300x250. At the end of the post, in various themes, two ads appear side by side horizontally.

But we could have the option of showing a single ad like this with the dimension of 600x250, the double of the ad I mentioned but horizontally.

And the same example serves for advertisements on the side, there could be a vertical advertisement with the dimension of 300x500, which is double the standard advertisement I mentioned above but in the vertical. And it's not so long as another one that Wordads also allows you to insert.

These would be options that would improve Wordads a little more, which is still limited.

**Comment by jeherve** [comment]
I would recommend that you purchase a paid WordPress.com plan to get more control over the ads displayed on your site. Once you've purchased a plan, you'll have access to the settings mentioned here:
https://wordpress.com/support/wordads-and-earn/#ads-settings

In addition to those global settings, you'll have access to Ad blocks inside the block editor; you'll be able to insert ads of the format you choose:
![image](https://user-images.githubusercontent.com/426388/106273443-f8fab980-6232-11eb-94eb-63008c30c229.png)

**Comment by ricjcs** [comment]
I have the paid plan and I don't have access to the options I mentioned. These would be options for improving the paid plan.

The block is not a solution because they have to be inserted individually in each post and pages. You can't automatically turn on/off for the entire site.

**Comment by jeherve** [comment]
> I don't have access to the options I mentioned.

You should be able to follow the instructions here to gain access to those options, that will apply to the whole site:
https://wordpress.com/support/wordads-and-earn/#applying-for-wordads

**Comment by ricjcs** [comment]
But those options that you mentioned I have. I have all the options that Wordads offers.
But the ones I mentioned do not exist

Regarding the first example I gave. It is possible to turn ads on or off on posts. But it is not possible to turn on or off those that only appear a few times inside the posts.

You cannot have a separate control of those that appear inside and below the posts. Turning off those that appear below the posts, turns off those that appear inside.

Regarding the sizes, I just suggested more options...

**Comment by ricjcs** [comment]
I can't turn off the ads inside the posts and keep the ads below the posts on, for exemple.

**Comment by ricjcs** [comment]
In these options that appear, there is no mention of the ads inside the posts. It says "Display ads below posts on: posts, pages, etc." 

But in reality it shows below and sometimes inside. And it is for those inside that I have also suggested an option for control.

**Comment by jeherve** [comment]
Thanks for the extra details.

> I can't turn off the ads inside the posts and keep the ads below the posts on

Right, I see what you mean. This is where the Ad block is useful, as it gives you more control; it allows you turn off ads for all posts, and then add ad blocks exactly where you want them, in each post.

This then ties in with your other issue, #18559, in how one can create block templates that will apply to all posts. I believe this should fit your needs best.

> Regarding the sizes, I just suggested more options...

I'm afraid 300x500 is not a very widely used ad size at the moment. The options available in the Ad block are amongst the most popular ad sizes, where we can ensure that we can provide ads from our ad netword, WordAds. We may add more sizes in the future, but for that to happen the sizes need to be popular enough that ad providers have creatives for them.

**Comment by ricjcs** [comment]
> This then ties in with your other issue, #18559, in how one can create block templates that will apply to all posts. I believe this should fit your needs best.

Yes it is also related, although the issue #18559 is related to what appears below the post, which is what wordads already allows. 

The wordads do not allow is the control of the ads that it automatically inserts inside. It either shows inside and below or it shows nothing. But yes, with the issue #18559 option, I could only use the blocks to control the ads, although that was not why I opened this issue.

Unfortunately issue #18559 was also closed, this option would allow more versatility in the use of blocks, not just for ads, but for other possibilities.. :)
